### ISSUE TITLE
Back End - Organization registry - accessibility

### ISSUE THREAD

**Issue by PamelaJohn** [voteamerica/voteamerica.github.io#353]
As a field organizer, I want to be able to view and manage lists of riders, drivers and organizations the campaign works with. 

#### Problem: The Current Database Font is Too Small
Volunteer organizers who are visually impaired are unable to read the database font.

**Comment by jkbits1** [comment]
@acqsysandrea @aaghevli following our meeting, just pinging about these as discussed

**Comment by jkbits1** [comment]
@PamelaJohn there is now an operator page which addresses this issue directly. If browser issues exist, please raise an issue for those 👍
